Gemfile

```diff
@@ -21,25 +21,23 @@ gem 'net-ldap'
 
 # Use puma as the webserver in development
 gem 'puma'
-gem 'rack', '1.6.4' # Remove in rails 5
```

Comment:
I think it was something which prevented puma from starting in the docker container. If you can get the service running in docker, it is probably fine